src/utils/parse-comment/index.js

```diff
@@ -75,20 +77,21 @@ function parseAddComment(doc, action) {
         })
         .data()[0].text
 
-    // TODO: handle plurals (e.g. some said docs)
-    let contributions = doc
-        .match('#Contribution')
+    const contributions = doc
+        .normalize({ plurals: true })
```

Comment:
Can we enable the test for this:
https://github.com/all-contributors/all-contributors-bot/blob/master/test/utils/parse-comment/index.test.js#L72

Reply:
Will od